changelogs/fragments/logstash.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
I'd classify this rather as a bugfix, since the message is kind of wrong (Python 2 is no longer supported).
```suggestion
bugfixes:
```